src/libData/AccountData/Account.h

```diff
@@ -114,13 +123,18 @@ class Account : public Serializable
     /// Computes an account address from a specified PubKey.
     static Address GetAddressFromPublicKey(const PubKey& pubKey);
 
+    /// Computes an account address from a sender and its nonce
+    static Address GetAddressForContract(const Address& sender,
+                                         const uint256_t& nonce);
+
     friend inline std::ostream& operator<<(std::ostream& _out,
                                            Account const& account);
 };
 
 inline std::ostream& operator<<(std::ostream& _out, Account const& account)
 {
-    _out << account.m_balance << " " << account.m_nonce;
+    _out << account.m_balance << " " << account.m_nonce << " "
```

Comment:
The naming convention here `_out` seems passing the past review but actually looks distinct in our codebase. Let's not introduce anything new and just follow the current style